docs/Installation.md

```diff
@@ -91,6 +90,19 @@ Zonemaster::CLI, see the [declaration of prerequisites].
    sv_SE.utf8
    ```
 
+#### Pre-built packages (Debian only)
+
+2) Configure Zonemaster repository
+   ```sh
+   curl -Ls https://package.zonemaster.net/setup.sh | sudo sh
+   ```
+3) Install Zonemaster CLI
+   ```sh
+   sudo apt install zonemaster-cli
+   ```
+
+#### CPAN Installation
```

Comment:
I think it would be better with
```
### Installation on Debian
Using pre-built packages is the preferred method for Debian. If you prefer to
install from CPAN instead, follow the steps for Ubuntu.

(Steps to install with packages)


### Installation on Ubuntu

(Steps to install from CPAN)
```

Reply:
I have updated like you suggested. I also moved the locales configuration after installation because on minimal environment (I am testing in debian/ubuntu container) the `locales` packages in not installed, I added it to the dependencies.

Reply:
I think it looks fine now except for that I am uncomfortable with the remote script. Is that only me?

Reply:
I have shared opinion on the subject, but I think it has become quite a popular pattern (even if insecure) for third party package installation. As stated in the other comment if that is better I can split the command in two parts (download and then execute, as opposed to piping to sh), which is a bit better in my opinion.

Reply:
In the end I think short and simple instructions are nice to have and can bring new users that don't want to deal with longer  or more complicated ones.

Reply:
Maybe we could let the user verify a checksum of the script? Would that improve the security situation enough to be worth the extra work for our users and ourselves?

Reply:
It maybe improves the security slightly, but I do not think it is worth it.